### ISSUE TITLE
Get more contributors.

### ISSUE THREAD

**Issue by deavmi** [FreeMusicFederation/license#6]
Tag all the friends (or followers, nvm, this is GitHub. Tag all the developers, and people that can rewrite posh English to make the license sound good but people who can make it also understandable at the same time.

@iamale
@Walkman100
@psgs
@gubuntu
@timlinux

**Comment by iamale** [comment]
What's the difference from various [Creative Commons](https://creativecommons.org/licenses/) flavours?

**Comment by deavmi** [comment]
Does the creative Commons force the source code to be published. I think no. Of what I know.

**Comment by iamale** [comment]
I see.

The problem is, you don't have a clear definition of a project file. I could call the resulting WAV a project file, for example. (And this could be the case, for example, if I made the whole waveform by hand, sample per sample)

**Comment by iamale** [comment]
This doesn't seem serious until somebody wants to remix the work. If they use different program, you're doomed, since they could take a _source-open_ track and make a binary remix, then say that it's the project file..

**Comment by iamale** [comment]
> The compiled project file must output the exact same, in terms of the binary, audio file

Did you know that even `gcc` does not produse the same output twice? That's for programs, in terms of music, there would be even more binary-level changes.

Also, this wording would prohibit transcoding (e. g. from FLAC to MP3), I believe.

**Comment by iamale** [comment]
A-a-another thing is that you probably don't want to get the source code each time you download a free track from the INTERNETS, so this should be expressed in the text as well.

**Comment by deavmi** [comment]
Well source code must be published. But that doesn't mean somebody that wants a track must download the source to listen to it. He can simply use the compiled exported audio file.

**Comment by deavmi** [comment]
I will make a better definition of a project file then. Is was planning to do so.

**Comment by deavmi** [comment]
@iamale Do you think adding a link like this would be okay?

````markdown
Copyright (c) The Crowbar Development Team [et al](https://github.com/FreeMusicFederation/license#credits). 2015
````

**Comment by iamale** [comment]
Not sure it is really needed. I suppose with GNU licenses, if you make a contribution, you assert that you're waiving your rights to your changes, so there is only one copyright holder and it is easier to defend your rights in court if somebody violates the license. At least this is the case with GPL.

You could make a file called `CONTRIBUTORS`, though -- this would be very nice. I don't think that putting it in `README` is a good idea, since the list may be quite huge

**Comment by deavmi** [comment]
@iamale Good point.

**Comment by iamale** [comment]
As a side point: don't you think that hiring a real lawyer would be better than writing _Legalese_ by hand? This is a task for pros, one single mistake could ruin the whole thing.

**Comment by deavmi** [comment]
@iamale Good point. Still, seems like a fun thing to do though.

**Comment by deavmi** [comment]
In a license must you state all aspects and how they are handled or just one thing for all.

**Comment by iamale** [comment]
> In a license must you state all aspects and how they are handled or just one thing for all.

Was that a question? :)

**Comment by deavmi** [comment]
@iamale Yep :smile:

**Comment by iamale** [comment]
@deavmi I'm not sure then :new_moon_with_face: Probably more detail is better, but then it becomes confusing.

**Comment by deavmi** [comment]
@iamale Acoording to people on reddit, the CC Share-Alike license is the same as this license, but I really doubt or see how the CC license makes sure that source code, not compiled data, source code is shared too.

**Comment by deavmi** [comment]
@iamale Yep more detail, but bearable, just explained well enough with good amount of detail.

**Comment by deavmi** [comment]
@iamale There is hope, (read the last comment). http://www.reddit.com/r/freesoftware/comments/321ooj/anybody/

**Comment by iamale** [comment]
In fact, I have a thought. Why don't you analyse some flavour of CC first? Some easy thing like [CC 1.0 BY-SA](https://creativecommons.org/licenses/by-sa/1.0/legalcode).

**Comment by iamale** [comment]
This would provide a better legal base for the license.

**Comment by deavmi** [comment]
@iamale I replaced the word creator, with licensor :smile:

**Comment by deavmi** [comment]
@iamale Yep, thanks. :smile: Will do :D

**Comment by deavmi** [comment]
@iamale Check this out - https://freemusicfederation.github.io
